### DISCUSSION TITLE
cannot use file_picker with serverpod

### DISCUSSION THREAD

**Discussion by adultviews20** [serverpod/serverpod#1604] [Q&A] [answered]
Hi, could someone help me, i try to make endpoint with a file_picker library
and got the error :  Error: Dart library 'dart:ui' is not available on this platform.
![image](https://github.com/serverpod/serverpod/assets/90047188/8cac2c44-d31f-4811-9d31-aa8209057fff)

**Comment by fleeser** [comment, marked as answer]
This is because you cannot pick files on the server side. You have to use the FilerPicker package inside your Flutter app.
https://docs.serverpod.dev/concepts/file-uploads
